classes/class-wc-connect-taxjar-integration.php

```diff
@@ -67,6 +67,11 @@ public function init() {
 		// Add toggle for automated taxes to the core settings page
 		add_filter( 'woocommerce_tax_settings', array( $this, 'add_tax_settings' ) );
 
+		// Fix tooltip with link on older WC.
+		if ( version_compare( WOOCOMMERCE_VERSION, '4.3.0', '<' ) ) {
+			add_action( 'admin_enqueue_scripts', array( $this, 'fix_tooltip_keepalive'), 11 );
+		}
+
```

Comment:
`4.3.0` beta is out, would https://github.com/woocommerce/woocommerce/pull/26558 made it in `4.3.0` in time? Otherwise, we might need to bump this version up.